src/Logging/Settings.hs

```diff
@@ -64,7 +64,12 @@ readLogDestination :: String -> Either String LogDestination
 readLogDestination = \case
   "stdout" -> Right LogDestinationStdout
   "stderr" -> Right LogDestinationStderr
-  x -> Right $ LogDestinationFile x
+  ('@' : path) -> Right $ LogDestinationFile path
```

Comment:
Cool. Is `@` a common convention?

Reply:
From commit / PR description,

> Prefixing file arguments with "@" is a common convention (see curl) we can leverage.

I thought aws did it too, but I think they actually prefer `file://<path>`, which I dislike because I don't know how many `/`s to put for a relative file.